apps/site/src/views/Resources/components/ResourceCard/ResourceCard.module.scss

```diff
@@ -21,18 +21,28 @@
 	justify-content: center;
 	align-items: center;
 	gap: 10px;
-	height: 450px;
+	height: 350px;
 }
 
 .tag {
-	position: absolute;
-	left: -5%;
-	bottom: 5%;
+	background-image: url("~@/assets/images/resource-link.svg");
+	background-size: 100% 100%;
+	background-repeat: no-repeat;
+	background-position: left;
+	display: inline-block;
+	padding: 1rem;
+	padding-right: 3rem;
+	margin-left: -6px;
+	margin-bottom: 0.5rem;
 	transition:
 		filter 0.13s ease,
 		transform 0.13s ease,
 		filter 0.13s ease;
 
+	color: black;
```

Comment:
Nitpick: primary text color is `theme.$black` which is slightly lighter than pure black.